ez/ez.go

```diff
@@ -27,6 +30,8 @@ type dialsOptions struct {
 	autoSetToSlice bool
 	flagSubstitute dials.Source
 	onWatchedError dials.WatchedErrorHandler
+	fileKeyDecoder caseconversion.DecodeCasingFunc
+	fileKeyEncoder caseconversion.EncodeCasingFunc
```

Comment:
nit: this `dialsOptions` struct is getting a bit large not to have any structure, comments or segmentation.

I wonder whether this option should operate internally as a slice of additional manglers. (then we don't need a separate `autoSetToSlice` field)

Reply:
We could just keep a slice of manglers, but it would be a little annoying to manage as `autoSetToSlice` is defaulted to true and disabling it would mean we'd have to loop through the slice to find the mangler and remove it.  We could use a map to make it easier to find, but then we'd have to just extract the values when it comes time to use them.  For the time being I've reorganized the fields in the `dialsOptions` struct so they make a little more sense.  If it gets more complicated in the future we can re-evaluate then as `dialsOptions` is private to the `ez` struct as it is... :-)